### DISCUSSION TITLE
Add option for criteria to select reference file (time / filename length)

### DISCUSSION THREAD

**Discussion by beng-tang** [hsoft/dupeguru#1151] [Ideas]
Hi, a suggestion:  Add an options to select the criteria to mark files as the reference file to keep, eg:
 Mark newest file as reference copy
 Mark oldest file as reference copy
 Mark longest filename as reference copy
 Mark shortest filename as reference copy

**Comment by RiseShineShake** [comment]
definitely want: Mark newest/oldest file as reference copy
